### ISSUE TITLE
Add tests to ensure the contrast ratios we promise are met

### ISSUE THREAD

**Issue by LeaVerou** [shoelace-style/webawesome#540]
We [currently promise that](http://localhost:4000/docs/tokens/color/):

> - A difference of 40 ensures a minimum 3:1 contrast ratio, suitable for large text and icons (AA)
> - A difference of 50 ensures a minimum 4.5:1 contrast ratio, suitable for normal text (AA) and large text (AAA)
> - A difference of 60 ensures a minimum 7:1 contrast ratio, suitable for all text (AAA)

Since this might be a legal requirement for folks, we should have a testsuite that reads our palettes and validates this.

Given that we have 11 hues, this means testing the following combinations:
- **5** combinations with a difference of **40**: 10/50, 20/60, 30/70, 40/80, 50/90
- **4** combinations with a difference of **50**: 10/60, 20/70, 30/80, 40/90
- **3** combinations with a difference of **60**: 10/70, 20/80, 30/90

I.e. 12 tests per hue.
If we want to also test APCA, this doubles the number of tests, since APCA distinguishes between foreground and background.
